src/main/java/org/shanerx/tradeshop/Utils.java

```diff
@@ -21,13 +21,13 @@
 
 package org.shanerx.tradeshop;
 
-import org.bukkit.Bukkit;
-import org.bukkit.ChatColor;
-import org.bukkit.Material;
+import org.bukkit.*;
```

Comment:
That was made by my IDE, didn't notice it. Wildcard imports are a *really* bad practice so I will fix this immediately.